app/zkvms/[slug]/page.tsx

```diff
@@ -51,7 +51,7 @@ export default async function ZkvmDetailsPage({
     return notFound()
   }
 
-  const activeClusters = await getActiveClusters()
+  const activeClusters = await getActiveClusters({ zkvmId: zkvm.id })
```

Comment:
missing filter for this page where we just need the clusters that are using the visited zkvm